### ISSUE TITLE
Disabling the auditable aspect should be configurable

### ISSUE THREAD

**Issue by kpagratis** [pmonks/alfresco-bulk-import#38]
We need to be able to take advantage of the Auditable Aspect behaviours on import.  Could this be configurable from the form?

**Comment by pmonks** [comment]
Yes, but in doing so you would lose the ability to control the created and modified dates.

**Comment by kpagratis** [comment]
That's fine.  I'm trying to rebuild the amp but am having issues.

**Comment by kpagratis** [comment]
The issue I'm having is that I believe that is affecting my code running on the onCreateNode event.

**Comment by kpagratis** [comment]
I'm having a bear of a time getting this AMP created.  I keep getting:
Failed to execute goal org.alfresco.maven.plugin:alfresco-maven-plugin:2.1.0:set-version (default-set-version) on project sample-bulk-import-source: Execution default-set-version of goal org.alfresco.maven.plugin:alfresco-maven-plugin:2.1.0:set-version failed: Unable to load the mojo 'set-version' in the plugin 'org.alfresco.maven.plugin:alfresco-maven-plugin:2.1.0' due to an API incompatibility: org.codehaus.plexus.component.repository.exception.ComponentLookupException: org/alfresco/maven/plugin/VersionMojo : Unsupported major.minor version 52.0

Which is annoying considering I'm not running JDK 8.0.  Any chance I could get an amp with line 159 commented out in amp/src/main/java/org/alfresco/extension/bulkimport/impl/BatchImporterImpl.java

That would be great.  kostas@pagratis.us would be fine

**Comment by pmonks** [comment]
> The issue I'm having is that I believe that is affecting my code running on the onCreateNode event.

That's possible if your behaviour is bound to the cm:auditable aspect (or a sub-aspect).  Are you able to bind to something else?

The version error you're seeing at build time is a bug in the Alfresco SDK: https://github.com/Alfresco/alfresco-sdk/issues/301
